### PR TITLE
v0.14.0 pre release changes

### PR DESCRIPTION
**Pull request by agoallikmaa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ Then, add the locally built library as a dependency in your application's build.
 ```
 dependencies {
 ...
-    implementation ("com.splunk:splunk-otel-android:0.14.0-SNAPSHOT")
+    implementation ("com.splunk:splunk-otel-android:0.14.0")
```
Comment (mateuszrzeszutek) [inline README.md:120]: This is about the local build, it's still going to be `0.14.0-SNAPSHOT`

**Review by mateuszrzeszutek — APPROVED** [review]
This needs to wait for https://github.com/signalfx/splunk-otel-android/pull/276 -- which in turn waits for the instrumentation 1.13.1 patch release.

File: README.md (hunk 1 above)
Comment (agoallikmaa) [reply README.md]: Changed back. On last release this was changed in pre and back in post, I guess that was unintended there then.

**Comment by mateuszrzeszutek** [comment]
@agoallikmaa please rebase with the main branch - https://github.com/signalfx/splunk-otel-android/pull/276 was merged, we can continue the release.

**Review by laurit — APPROVED** [review]

**Pull request merged**